apps/explorer/src/pages/transaction-result/transaction-summary/index.tsx

```diff
@@ -11,29 +11,51 @@ import {
 import { BalanceChanges } from './BalanceChanges';
 import { ObjectChanges } from './ObjectChanges';
 
+import { LoadingSpinner } from '~/ui/LoadingSpinner';
+import { Text } from '~/ui/Text';
+
 interface TransactionSummaryProps {
     transaction: SuiTransactionBlockResponse;
 }
 
 export function TransactionSummary({ transaction }: TransactionSummaryProps) {
-    const summary = useTransactionSummary({
+    const summaryData = useTransactionSummaryWithNFTs({
         transaction,
     });
 
     const transactionKindName = getTransactionKindName(
         getTransactionKind(transaction)!
     );
 
-    const balanceChanges = summary?.balanceChanges;
-    const objectSummary = summary?.objectSummary;
+    const balanceChanges = summaryData?.balanceChanges;
+    const objectSummary = summaryData?.objectSummary;
+    const objectSummaryNFTData = summaryData?.objectSummaryNFTData;
```

Comment:
It feels wrong to have this so separated vs just inline with the object summary, what's the purpose of that?